connection.go

```diff
@@ -207,26 +212,39 @@ type Opts struct {
 // Address could be specified in following ways:
```

Comment:
Is it possible to make a list of lists in go doc? Because this is exactly the situation we have here.

Maybe the best way to organize this is something like

Address could be specified in following ways:
- TCP connections (`tcp://192.168.1.1:3013`, `tcp://my.host:3013`, ...)
- UNIX sockets (`unix:///abs/path/tnt.sock`, `unix:path/tnt.sock`, ...)

Reply:
Personally, I don't like it. This way it looks ugly too.

```diff
@@ -203,37 +204,18 @@ type Opts struct {
 	Logger Logger
 }
 
-// Connect creates and configures new Connection
+// Connect creates and configures a new Connection.
 //
 // Address could be specified in following ways:
 //
-// TCP connections:
-//
-// - tcp://192.168.1.1:3013
-//
-// - tcp://my.host:3013
-//
-// - tcp:192.168.1.1:3013
-//
-// - tcp:my.host:3013
-//
-// - 192.168.1.1:3013
-//
-// - my.host:3013
-//
-// Unix socket:
-//
-// - unix:///abs/path/tnt.sock
-//
-// - unix:path/tnt.sock
-//
-// - /abs/path/tnt.sock  - first '/' indicates unix socket
-//
-// - ./rel/path/tnt.sock - first '.' indicates unix socket
+// - TCP connections (tcp://192.168.1.1:3013, tcp://my.host:3013,
+// tcp:192.168.1.1:3013, tcp:my.host:3013, 192.168.1.1:3013, my.host:3013)
 //
-// - unix/:path/tnt.sock - 'unix/' acts as a "host" and "/path..." as a port
+// - Unix socket, first '/' or '.' indicates Unix socket
+// (unix:///abs/path/tnt.sock, unix:path/tnt.sock, /abs/path/tnt.sock,
+// ./rel/path/tnt.sock, unix/:path/tnt.sock)
 //
```

![image](https://user-images.githubusercontent.com/1151557/163360719-b3f87047-b4ca-4e17-b353-be43e4479871.png)

Reply:
Yeah, it is a bit ugly. Choose the one you're prefer.

Reply:
Left it as is (two items with 'TCP' and 'Unix' connection).